src/components/home/PanelResult.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <v-card outlined>
+    <v-card-text>
+      <v-data-table
+        :headers="tableHeaders"
+        :items="getPanelResult()"
+        item-key="name">
+        <template v-slot:top>
+          <v-toolbar flat>
+            <v-toolbar-title>{{ $t('panel-result.result.name') }}<span v-if="userGenes.length > 0">({{ $t('panel-result.result.title') }}: {{ userGenes.length }})</span></v-toolbar-title>
+          </v-toolbar>
+          <v-dialog v-model="showDialog" persistent max-width="800px">
```

Comment:
Dialog should be at the root of template (sibling to v-card) since it's not really "inside" the table.

Reply:
I tried doing that but vue cli throws an error saying "Component template should contain exactly one root element. If you are using v-if on multiple elements, use v-else-if to chain them instead"

Reply:
Wrapping both elements into a div at the root of the template should do the trick. Vue 3 got rid of the 1 child rule but it's not quite ready for production yet so we're sticking with Vue 2

Reply:
done